### ISSUE TITLE
Errors during some of the Crane Kung Fu attacks

### ISSUE THREAD

**Issue by x-qq** [CleverRaven/Cataclysm-DDA#59600]
### Describe the bug

Some of the Crane Kung Fu attacks currenty produce errors, I think it's the Crane Strike.

### Steps to reproduce

1. Make a new character with "Shaolin Kung Fu" background and select Crane Kung Fu.
2. Level unarmed to 5, switch to Crane Kung Fu style.
3. Spawn some zombies and dodge an attack so that "Crane's Flight" OnDodge buff activates.
4. Try hitting things and observe errors:
```
 DEBUG    : invalid body part id "hand_wrist_l"

 FUNCTION : int_id<T> generic_factory<T>::convert(const string_id<T>&, const int_id<T>&, bool) const [with T = body_part_type]
 FILE     : src/generic_factory.h
 LINE     : 477
 VERSION  : ad3fd73
```

### Expected behavior

I expect to have no errors

### Versions and configuration

Game version: cdda-linux-tiles-x64-2022-07-22-0050
Mod list: default

### Additional context

Error looks very similar to #59559

**Comment by Trioct** [comment]
It should've been fixed in #59570. Looking at your game version I'm assuming you last updated 7-22, or 3 days ago. Can you try updating?

**Comment by x-qq** [comment]
You are right, it's fixed.
